pkg/segment/query/segquery.go

```diff
@@ -1317,3 +1319,41 @@ func applyQsrsFilterHook(qsrs []*QuerySegmentRequest, isRotated bool) ([]*QueryS
 
 	return qsrs, nil
 }
+
+func getSearchTermsForNode(sNode *structs.SearchNode, searchTerms map[string][]string) {
```

Comment:
nit: rename to `addSearchTermsForNode` since `searchTerms` is the input and the "output"